core/components/molecules/keyValuePair/ValueElement.tsx

```diff
@@ -0,0 +1,44 @@
+import * as React from 'react';
+import { Text } from '@/index';
+import classNames from 'classnames';
+import { BaseProps, extractBaseProps } from '@/utils/types';
+
+export interface ValueElementProps extends BaseProps {
+  /**
+   * React Element to be added inside `ValueElement`
+   */
+  children?: React.ReactNode;
+  /**
+   * Specify value to be displayed in `ValueElement`
+   */
+  value?: React.ReactText;
+}
+
+export const ValueElement = (props: ValueElementProps) => {
+  const { children, value, className } = props;
+
+  const baseProps = extractBaseProps(props);
+
+  const valueClassNames = classNames(
+    {
+      ['m-0']: true,
+    },
```

Comment:
Using square brackets is not required. You can also pass it directly as ('m-0', className).